Sources/SourceKitLSP/SourceKitLSPServer.swift

```diff
@@ -2143,7 +2143,7 @@ extension SourceKitLSPServer {
           return CallHierarchyIncomingCall(
             from: indexToLSPCallHierarchyItem(
               symbol: related.symbol,
-              moduleName: definitionSymbolLocation?.moduleName,
+              containerName: definition?.containerName,
```

Comment:
Should module be the fallback if there is no container?

Reply:
No, I think it’s better to leave the `containerName` empty for top-level functions. That way it’s also easier to tell top-level functions apart from member functions.